lib/memstream.c

```diff
@@ -258,6 +349,26 @@ KTX_error_code ktxMemStream_read(ktxStream* str, void* dst, const ktx_size_t cou
     return KTX_SUCCESS;
 }
 
+/**
+* @~English
+* @brief A function used to deny read operations on a ktxMemStream.
+*
+* @param [in]     str      pointer to ktxMem struct, converted to a void*, that
+*                          specifies an input stream.
+* @param [in,out] dst      pointer to memory where to copy read bytes.
+* @param [in,out] count    pointer to number of bytes to read.
+*
+* @return      KTX_INVALID_OPERATION.
+*/
+static
+KTX_error_code ktxMemStream_disable_read(ktxStream* str, void* dst, const ktx_size_t count)
+{
+    UNUSED(str);
```

Comment:
What is the point of this function? It does nothing.

A simpler fix  for unused parameters is to remove the parameter name leaving only the type. `UNUSED` is intended for situations such as when there is a variable only used for an assert so unused in a Release build.

Reply:
that's quite literally your idea
https://github.com/KhronosGroup/KTX-Software/discussions/614#discussioncomment-3354432
> The read function for such a stream should return KTX_INVALID_OPERATION.

Reply:
> A simpler fix for unused parameters is to remove the parameter name leaving only the type. `UNUSED` is intended for situations such as when there is a variable only used for an assert so unused in a Release build.

Okay I rushed things a bit and edited files without trying them out first. Turns out the alternative you suggest is invalid in C and fails to build with MSVC (error code C2055). I'll be reverting the changes.

Reply:
> that's quite literally your idea

I was confused by the function name which implies that it should be doing something, such as setting a flag, to disable read. Maybe `ktxMemStream_no_read. I hope to do a full review of this early next week.

Sorry for the bad steer on the parameter names.